pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -289,20 +288,15 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 		qosContainerManager: qosContainerManager,
 	}
 
-	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.TopologyManager) {
-		cm.topologyManager, err = topologymanager.NewManager(
-			machineInfo.Topology,
-			nodeConfig.ExperimentalTopologyManagerPolicy,
-			nodeConfig.ExperimentalTopologyManagerScope,
-			nodeConfig.ExperimentalTopologyManagerPolicyOptions,
-		)
-
-		if err != nil {
-			return nil, err
-		}
+	cm.topologyManager, err = topologymanager.NewManager(
```

Comment:
looking inside the NewManager() impl, should the calls to NewPolicyOptions and NewNUMAInfo calls be skipped if the policy is PolicyNone? Currently, it looks like we can fail to start kubelets if we can't discover NUMA topology, even though that won't be used at all if the policy is PolicyNone

Reply:
This is a valid point. Previously users had the chance to exclude this flow with the FG, this option is now being removed. My 2c: this predates the GA graduation and could require some code massaging (e.g. not a simple if) so IMO is best addressed with a followup PR, which should be tracked (but probably not blocking) in the GA-graduation issue.  I'll let @klueska (cc @SergeyKanzhelev ) make the final call.

Reply:
https://github.com/kubernetes/kubernetes/issues/116391

Reply:
thanks @SergeyKanzhelev !